python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
-		"processType": "directSupplierIncident",
-		"aptCommentBox":{
-			"aptComment":{
-				"commentText": "Please provide a list of missing items and we will replace them.",
-				"visibilityType": "Public"
-				}
-			}
-		}
+comment_request = {
+  "header": {
+    "appName": "agile-process-teams",
+    "dataspace": "default",
+    "eventName": "agile-process-teams:add-comment-for-incident:v1",
+    "headerVersion": 1,
+    "ownerId": "YOUR_OWNER_ID",
+    "processNetworkId": "YOUR_PROCESS_NETWORK_ID"
+    },
+  "payload": {
+    "processId": "my_incident_id",
+    "processType": "directSupplierIncident",
+    "aptCommentBox":{
+      "aptComment":{
+        "commentText": "Please provide a list of missing items and we will replace them.",
+        "visibilityType": "Public"
+      }
+    }
+  }
 }
 ```
+Remember that we created a dictionary object with the above data.  If we wish to update the `ITEM_ID` with the result from the previous step, we can use the `.update()` method.  
+Let's make our modifications and add a comment.  
 
-Do not worry that your `processId` does not match the example. You can load the payload comment file and update the `processId` number dynamically. Enter the following in your Python terminal:
-```
->>> comment_payload_file = 'comment_payload.json'
->>> comment_payload = utils.read_payload_file(comment_payload_file)
->>> comment_payload
-{'processId': '4ec1229a-d46f-4fc0-878f-188d318db55a', 'processType': 'directSupplierIncident', 'aptCommentBox': {'aptComment': {'commentText': 'Please provide a list of missing items and we will replace them.', 'visibilityType': 'Public'}}}
-```
-Your payload has been loaded properly, but the `processId` is still wrong. Use the `update()` function to change the stored value:
-```
->>> comment_payload.update({"processId": "fd04dd70-c162-43ef-8118-539ca01de471"})
->>> comment_payload
-{'processId': 'fd04dd70-c162-43ef-8118-539ca01de471', 'processType': 'directSupplierIncident', 'aptCommentBox': {'aptComment': {'commentText': 'Please provide a list of missing items and we will replace them.', 'visibilityType': 'Public'}}}
-```
-You can use the same process to change the `eventName`:
+**Type or copy the following into your Python terminal**  
 ```
->>> event_data.update({"eventName": "agile-process-teams:add-comment-for-incident:v1"})
->>> event_data
-{'eventName': 'agile-process-teams:add-comment-for-incident:v1', 'ownerId': '2aaeed40-3912-427e-a94f-51fd5a730345', 'processNetworkId': '61077cb2-601b-4c27-b11f-5e3910589931'}
+comment_request.get('header').update({'ownerId': owner_id})
+comment_request.get('header').update({'processNetworkId': network_id})
+
+comment_request.get('payload').update({'processId': incident_id})
 ```
-The payload and event data should now be updated. You can verify this by resending the commands to build the request body and resend the request.  
+After making our desired modifications, we can convert our dictionary to a JSON string and send our request.  
```

Comment:
"After making the desired modifications, convert the dictionary to a JSON string and send the request."